src/main/java/com/epam/eco/commons/jdbc/ConnectionTimeoutWrapper.java

```diff
@@ -0,0 +1,61 @@
+/*
+ *Copyright 2020 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License.  You may obtain a copy
+ * of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.epam.eco.commons.jdbc;
+
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Method;
+import java.lang.reflect.Proxy;
+import java.sql.Connection;
+import java.sql.Statement;
+
+import org.apache.commons.lang3.Validate;
+
+/**
+ * @author Uladzislau_Belykh
+ */
+final class ConnectionTimeoutWrapper {
```

Comment:
1 It's not a wrapper, it's a proxy. Please use QueryTimeoutConnectionProxy name
2 Why it's package private?